prov/net/src/xnet_ep.c

```diff
@@ -113,8 +113,16 @@ static void xnet_set_no_port(SOCKET sock)
 
 int xnet_setup_socket(SOCKET sock, struct fi_info *info)
 {
-	int ret, optval = 1;
+	int ret, optval;
 
+	/* Request 128k buffer sizes, but use system defaults on failure */
+	optval = 128 * 2014;
```

Comment:
What magic number is '2014' based on? Typo of 1024?

Reply:
Yes, this should have been 128k.  This is the value used by Intel MPI 2018, so I went with it.  (2018 shows better tcp performance and scaling than later versions.)  4M per connection seems huge at scale.  With 1000 ranks, that's 4G per process, and we're currently running with about 56 ranks per node.  I can drop this patch and continue to use the system defaults.